Gemfile

```diff
@@ -71,6 +71,10 @@ group :development do
   gem 'annotate', :git => 'git://github.com/ctran/annotate_models.git'
   gem 'spring'
   gem 'rubocop', require: false
+
```

Comment:
Remove empty spacing and comment

Reply:
Thanks, by empty spacing should it instead look like this?

`group :development do
  gem 'annotate', :git => 'git://github.com/ctran/annotate_models.git'
  gem 'spring'
  gem 'rubocop', require: false

  # Used for sending emails to the browser
  gem 'letter_opener'
end`

Reply:
I would just remove the comment and spacing entirely

Reply:
Ah thanks I understand.